floor/floor/controller/layout.py

```diff
@@ -1,15 +1,29 @@
 from builtins import object
 import json
+import os
 
 
 class Layout(object):
-    def __init__(self, config_filename):
+    DEFAULT_CONFIG_NAME = 'floor-layout'
+
+    def __init__(self, config_dir, config_name=None):
+        self.config_dir = config_dir
         self.squares = None
         self.rows = None
         self.cols = None
         self.layout = None
 
-        with open(config_filename) as json_data:
+        if not config_name:
+            config_name = self.DEFAULT_CONFIG_NAME
+
+        self.load_config_from_name(config_name)
+
+    def load_config_from_name(self, name):
```

Comment:
The 'name' here is 'one-pod-layout' which will expand to the one-pod-layout.json file added above and in the config dir.